package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.3",
   "homepage": "https://github.com/aframevr/aframe-core",
   "license": "MIT",
-  "main": "src/index.js",
+  "main": "dist/aframe-core.js",
```

Comment:
> Not as worried about documenting `npm link` as that won't be too commonly done.

I'm worried about it. how do we dev against libraries ensuring things don't break? as mentioned several times in #699, I'm just asking for the commands and updated documentation.